### PR TITLE
Added supportsWindowFunctions() to replace env variable

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

File: src/Database/Driver/Sqlite.php [hunk 1]
```diff
@@ -62,6 +62,20 @@ class Sqlite extends Driver
...
+     * The connected server version.
+     *
+     * @var string
+     */
+    protected $_version;
+
+    /**
+     * Whether or not the connected server supports windows.
+     *
+     * @var bool
```
Comment (dereuromark) [inline src/Database/Driver/Sqlite.php:75]: Those seem to be |null here

File: src/Database/Driver/Mysql.php [hunk 2]
```diff
@@ -75,6 +75,13 @@ class Mysql extends Driver
      */
     protected $_supportsNativeJson;
 
+    /**
+     * Whether or not the connected server supports windows.
+     *
+     * @var bool
+     */
+    protected $_supportsWindows;
```
Comment (ADmad) [inline src/Database/Driver/Mysql.php:83]: Can we name this `$_supportsWindowFunctions` instead?

Comment (dereuromark) [reply src/Database/Driver/Mysql.php:83]: I was also like "wuuut?" :P

Comment (othercorey) [reply src/Database/Driver/Mysql.php]: Sure. I was just using the technical name for the expression, but can be confusing.

**Comment by markstory** [comment]
Nice work :clap:

**Pull request merged**